### ISSUE TITLE
Custom Data Viewer Layout names should be displayed

### ISSUE THREAD

**Issue by tatiang** [thunkable/thunkable-issues#969]
**Is your feature request related to a problem? Please describe.**
When you save a custom data viewer layout, it asks for a name for the layout. But this name doesn't appear in the list when later selecting a layout.

**Describe the solution you'd like**
The layout name should appear in the list of DVL layout choices. Screenshot for reference below. The name could be displayed to the left or right of the layout as it is currently shown.

**Describe alternatives you've considered**

**Which platform are you asking for?**
Thunkable X

**Additional context**
![image](https://user-images.githubusercontent.com/10355498/121951415-be8c4780-cd0f-11eb-922d-d868871c24ac.png)

**Comment by domhnallohanlon** [comment]
Noted, thanks!

**Comment by tatiang** [comment]
I still don't understand why we are given the option to name a DVL layout if we can't access that name from within the project. It would be so much better to be able to see the name associated with each layout when picking a layout. Because small variations often look nearly identical graphically but with a name, we could explain the difference.

**Comment by Muneeralshaikh** [comment]
One difficulty I face when using Custom Layout is trying different options in the layout to finally choose one and this produces a number of very similar layouts to choose from. The lack of the name being displayed with the layout makes it so difficult to deal with.

Another issue, is that if the layout does not have visual components then it is not presented in the list in the project or presented with an empty rectangle. Having a name on the side or on top of it will help better identifying it.

**Comment by tatiang** [comment]
I hope this can be improved. Currently, if I'm viewing the list of custom layouts on the Projects page, I can see layout names but no components. It would be great to have a thumbnail view of each layout to connect to the name listed there. And if I'm viewing the list of custom layouts within a project after clicking on the layout property in the Design tab, I can see components/thumbnail views but... no names! It would be great to have names listed above each layout.

The trick right now is that I have no easy way to figure out which name goes with which layout!

**Comment by kartiktambi** [comment]
@domhnallohanlon

I agree with @tatiang & @Muneeralshaikh here, this would be a very helpful tweak in the existing list item layout options menu of DVLs as it gets really tough to keep track of slight variations/similar looking designs without the labels.

As of today we still haven't received any updates from Thunkable's side - kindly take this feature request into consideration & let us know here if there are any updates on this in the future. 👍
